### PR TITLE
Constants.scss  added

### PR DESCRIPTION
**Pull request by LenaESoftwire**

### CODE REVIEW

File: WhaleSpotting/react-app/src/styles/Constants.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap%27');
+
+$primaryDark:  #004785;
+$primaryColour: #0072B5;
+$primaryLight: #56A0E8;
+$secondaryColour: #CDE9FF;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Constants.scss]: This is the 4th primary

File: WhaleSpotting/react-app/src/styles/Constants.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+$mobileTitleFontSize: 24px;
+$mobileSubtitleFontSize: 20px;
+$mobileBodyFontSize: 14px;
+$smallMobileTitleFontSize: 18px;
+$smallMobileSubtitleFontSize: 14px;
+$smallMobileBodyFontSize: 10px;
+
+$bodyWidth: 1024px;
+$activeImageBorder: 10px solid $primaryColor;
+$cardBackgroundColour: $secondaryDark;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Constants.scss]: Wrong colour

File: WhaleSpotting/react-app/src/styles/Constants.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+$primaryDark:  #004785;
+$primaryColour: #0072B5;
+$primaryLight: #56A0E8;
+$secondaryColour: #CDE9FF;
+$secondaryLight: #FFFFFF;
+$secondaryDark: #9BB7CC;
+$textOnPrimary: #FFFFFF;
+$textOnSecondary: #000000;
+
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Constants.scss:12]: Can we add the red colour

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Id either remove a load of these constants as we aren't sure they should be those, or they should be grouped and renamed to a more conssitent format. e.g. bodyWidth is a lot of lines from desktopContentWidth, and seem like they should be related?

File: WhaleSpotting/react-app/src/styles/Constants.scss (hunk 1 above)
Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/Constants.scss]: Called #primaryCard

File: WhaleSpotting/react-app/src/styles/Constants.scss (hunk 2 above)
Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/Constants.scss]: fixed

File: WhaleSpotting/react-app/src/styles/Constants.scss (hunk 3 above)
Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/Constants.scss:12]: added

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**